### PR TITLE
New package: Prog.ProgFinder version 2.39

### LINKED ISSUE #154508: [Package Request]: Prog Finder
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.progdvb.com/prog_sat_finder.html
Publisher: Prog
Package Name: Prog Finder
Description: Simple and completely free the utility capable considerably to help with adjustment of the Satellite dish or Terristeral antenna.
Package Version: 2.39
Installer URL: https://files.progdvb.com/progfinder/ProgFinderSetup2.39x64.exe

### PR DESCRIPTION
**Pull request by schembriaiden**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.1 :rocket:

Resolves: https://github.com/microsoft/winget-pkgs/issues/154508
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154576)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 881.)

File: manifests/p/Prog/ProgFinder/2.39/Prog.ProgFinder.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: Prog.ProgFinder
+PackageVersion: '2.39'
+InstallerType: nullsoft
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2019-03-09
+Installers:
```
Comment (stephengillie) [inline manifests/p/Prog/ProgFinder/2.39/Prog.ProgFinder.installer.yaml:12]:
```suggestion
Dependencies:
  WindowsFeatures:
    - netfx3 
Installers:
```

**Comment by stephengillie** [comment]
Verify with dependency: 
image

**Comment by vikingnope** [comment]
I am getting this error when trying it with the latest dependency:

image

**Comment by vikingnope** [comment]
Ah yes I think I know why this happened on my machine, I am using windows sandbox, can you test the manifest as it currently is on your windows 10 vm @stephengillie, please?

**Comment by stephengillie** [comment]
> Ah yes I think I know why this happened on my machine, I am using windows sandbox, I believe you cannot enable windows features on there. Can you test the manifest as it currently is on your windows 10 vm @stephengillie, please?

Sure: 

> Verify with dependency: image
> 
> Edit: Does this need a `DisplayVersion`?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**